UM-USElab/gradecraft-development
# Free the learning objectives into the wild!

### Status
**READY**

### Description
This PR removes the admin constraint on Learning Objectives and allows instructors to turn them on independently. 

This PR removes a column from the courses table: `allows_learning_objectives` 

### Todos


### Deploy Notes
We should make sure to add documentation about use as well as start talking to instructors interested in them. 

https://gradecraft.zendesk.com/hc/en-us/articles/360038796653-Using-Learning-Objectives-in-Your-Course-Instructor-Guide-

### Migrations
YES

Developers will need to run ` bundle exec rails db:migrate ` and ` bundle exec rails db:migrate RAILS_ENV=test `

### Steps to Test or Reproduce
As an instructor, navigate to Course Settings and turn on your learning objectives card. Then confirm that you have a new sidebar item, and can attach learning objectives to assignments. 

### Impacted Areas in Application
List general components of the application that this PR will affect:

* Assignments, Course Settings, Student Dashboard
